site/content/in-dev/unreleased/metastores.md

```diff
@@ -32,6 +32,29 @@ In order to add other JDBC drivers, you have to build Polaris using the `eclipse
 
 ## Polaris Server Configuration
 
+### Relational JDBC
+Configure the `polaris.persistence` section in your Polaris configuration file
+(`application.properties`) as follows:
+
+```
+polaris.persistence.type=relational-jdbc
+
+quarkus.datasource.db-kind=postgresql
+quarkus.datasource.username=<your username>
+quarkus.datasource.password=<your password>
+
+quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/<database_name>
+```
+
+The relational JDBC metastore currently relies on a Quarkus-managed datasource and supports only Postgres and H2—similar to EclipseLink. However, we only provide documentation for using it with Postgres. In future releases of Apache Polaris, we plan to extend support to additional RDBMS datasources such as MySQL and MariaDB. These are not currently supported due to differences in schema requirements and error handling conventions.
```

Comment:
Can we just say only Postgres and H2 are supported in Polaris due to the schema differences? We may not put a plan in a usage doc.